test-scripts/README.md

```diff
@@ -0,0 +1,51 @@
+# integration-adaptor-111
+National Integration Adaptors - Adaptor for NHS 111 Post Event Message
+
+## How to use this image
+
+### Requirements:
+1. JDK 14
+2. Docker
+
+### Clone the repository
+
+`$ git clone https://github.com/nhsconnect/integration-adaptor-111.git`
+
+### Find the scripts folder
+
+`$ cd integration-adaptor-111/test-scripts`
+
+Each release has its own folder, use the scripts for specific release
+
+`$ cd 0.1.0`
+
+### Start it up
+
+`integration-adaptor-111/test-scripts/0.1.0$ ./run.sh`
+
+This will build the application container locally and start it along with ActiveMQ
+The services will listen on following ports
+
+* REST service `localhost:8080`
+* ActiveMQ: `localhost:5672`
+
+## Testing
+
+## How to run unit tests:
```

Comment:
This isn't valid for testing the release. It should only mention using the test scripts in the test-scripts/{version}/tests folders. Similar to this https://hub.docker.com/repository/docker/nhsdev/nia-nhais-adaptor

Reply:
So remove the test and integrationTest and leave only the acceptance test?

Reply:
These are tests used by the test partners to test the docker images running locally. Test partners won't run the unit/integration/acceptance tests, they will just run the shell scripts - like https://github.com/nhsconnect/integration-adaptor-111/pull/59/files#diff-f195099d292a84b4ecad3b94960e5313